core/domain/question_services_test.py

```diff
@@ -465,7 +466,8 @@ def _mock_logging_function(msg, *args):
             observed_log_messages.append(msg % args)
 
         logging_swap = self.swap(logging, 'error', _mock_logging_function)
-        assert_raises_context_manager = self.assertRaises(Exception)
+        assert_raises_context_manager = self.assertRaisesRegexp(
+            Exception, '\'unicode\' object has no attribute \'cmd\'')
```

Comment:
Ditto this is the message I got when ran backend test with a random string.

Reply:
If you found error while adding these checks then you'll have to at least look into it, try to solve it if you aren't able to do it ask anyone for help (use git blame to check the author and ask them politely for help). These workarounds won't help, if you don't know what's to do for these kinda case then raise a comment before assigning reviews so that they can reply/help/suggest to your comment while reviewing it instead of they spending time to understand it, raising questions and wait for follow up! (This will help your PR to get merged quickly!)

Reply:
I don't think we should raise this kinda errors, @aks681 Do you think we should have a type check in the update_question method?

Reply:
I think this is expected. We have a 'try.. catch' in `apply_change_list`, where if the changelist isn't valid, the exception is caught and printed, so the error case is taken care of in normal usage.

Also @Hudda, when a review comment is there asking the reason behind something, try to dig into the origin of the error message and then explain a bit more why you think the error is coming. Just telling, this is what is used elsewhere with no reason for it isn't the right way to go, just a tip!